moon/docker/rover/Dockerfile

```diff
@@ -245,6 +245,10 @@ RUN chmod 777 /rover_workspace
 
 RUN rm -rf /rover_workspace/devel
 RUN rm -rf /rover_workspace/build
+RUN rm -rf /osgar/.git
```

Comment:
why this? After this change it will be not possible to easily update and test changes?

Reply:
I don't want to be distributing my git credentials and personal setup in dockers that will go to NASA and such. What is the scenario where you think it's needed there? If you like to have git access from within the docker, it should be your git account there.

Reply:
good point - I am forgetting that this will no longer be for us only ...